third_party/move/move-compiler/src/expansion/translate.rs

```diff
@@ -1649,6 +1690,10 @@ fn access_specifier_list_as_acquires(
                             && check_wildcard(context, second)
                             && check_wildcard(context, third)
                     },
+                    NameAccessChain_::Four(..) => {
```

Comment:
Can we get a test for this code?  Or do we have one already?

Reply:
I found one here: third_party/move/move-compiler-v2/tests/checking/variants/variants_check_err.move